pydoctor/sphinx_ext/build_apidocs.py

```diff
@@ -9,8 +9,12 @@
   - C{pydoctor_args} - an iterable with all the pydoctor command line arguments used to trigger the build.
                      - (private usage) a mapping with values as iterables of pydoctor command line arguments.
 
+  - C{pydoctor_main_branch} - The name of the main branch name. Used on RTD for latest build.
+  - C{pydoctor_debug} - C{True} if you want to see extra debug message for this extension.
+
 The following format placeholders are resolved for C{pydoctor_args} at runtime:
   - C{{outdir}} - the Sphinx output dir
+  - C{{git_reference}} - the Git reference that can be used for source code links.
```

Comment:
Maybe call it `source_revision` or something similar? Since it doesn't have to originate from `git` when using RTD.

Reply:
I went with source_reference